protocols/relay/tests/lib.rs

```diff
@@ -521,6 +527,9 @@ async fn wait_for_reservation(
 
     loop {
         match client.select_next_some().await {
+            SwarmEvent::ExternalAddrConfirmed { address } => {
+                assert!(address.iter().any(|p| matches!(p, Protocol::P2pCircuit)))
```

Comment:
Why not assert the actual address here.

Reply:
Also, lets extend this test to ensure we only emit `ExternalAddrConfirmed` if `renewal` is false.

Reply:
> Why not assert the actual address here.

Oversight on my part.